changed the default of bootstrap_repo_flush to off (#597)
uyuni-project/uyuni-docs: modules/client-configuration/pages/bootstrap-repository.adoc

.Flush Mode

> By default, an exiting repository is only updated with the latest packages.

Reviewer: ```suggestion
By default, existing repositories are updated only with the latest packages.
```

> It can be configured to always start with an empty repository.

Reviewer: ```suggestion
You can configure it to always start with an empty repository instead.
```

If you are creating bootstrap repositories that include multiple different architectures, you need to be careful that all architectures are updated correctly.
For example, the {x86_64} and {ibmz} architectures for {slea} use the same bootstrap repository URL at ``/srv/www/htdocs/pub/repositories/sle/15/2/bootstrap/``.

> When the ``flush`` option is enabled, and you attempt to generate the bootstrap repository for multiple architectures, only one architecture would be generated.

Reviewer: ```suggestion
When the ``flush`` option is enabled, and you attempt to generate the bootstrap repository for multiple architectures, only one architecture is generated.
```
